### PR TITLE
autorandr: add rotate option

### PR DESCRIPTION
**Pull request by tuxinaut**
Add rotate option for `autorandr`

### CODE REVIEW

File: modules/programs/autorandr.nix [hunk 1]
```diff
@@ -72,6 +72,13 @@ let
         default = "";
         example = "1.0:0.909:0.833";
       };
+
+      rotate = mkOption {
+        type = types.string;
```
Comment (rycee) [inline modules/programs/autorandr.nix]: From the xrandr man page I get the impression that rotate only can have the values 'normal', 'left', 'right', or 'inverted'. If that's the case then it's better to have
```nix
type = types.enum [ "normal" "left" "right" "inverted" ]
```

**Comment by rycee** [comment]
Thanks for the pull request! I just added a suggestion about the option's type.

**Comment by tuxinaut** [comment]
I updated my pull request regarding your proposal :)

I'm new to `Ǹix`, So thx for the **enum** hint :+1:

**Comment by rycee** [comment]
Great! I've rebased the change to master in 6aa44d62ad6526177a8c1f1babe1646c06738614. Thanks for the contribution 🙂

**Pull request closed without merging**